frontend/src/pages/article/components/article-view/article-view.tsx

```diff
@@ -107,12 +106,16 @@ const ArticleView: React.FC<Properties> = ({
               iconClassName={styles.icon}
               onClick={onButtonClick}
             />
-            <IconButton
-              iconName="comment"
-              className={styles.iconButton}
-              iconClassName={styles.icon}
-              onClick={onButtonClick}
-            />
+            <Link
+              to={`#${LinkHash.COMMENTS}` as typeof AppRoute.ARTICLES_$ID}
+              state={article}
+            >
+              <IconButton
+                iconName="comment"
+                className={styles.iconButton}
+                iconClassName={styles.icon}
+              />
```

Comment:
If it's a link for now - then you don't need to use IconButton - it could be just an Icon